src/components/Modals/SelectModals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -90,7 +91,11 @@ export const SelectTokenModal: React.FC<ISelectTokenModal> = ({
             {tokens
               ? tokens
                   .filter(token => {
-                    return token ? token.symbol.toLowerCase().includes(value) : null
+                    return token
```

Comment:
maybe simplify this to: token.symbol.toLowerCase().includes(value) || token.name.toLowerCase().includes(value)